app/src/main/java/com/example/pictureupload/ui/screens/landing_screen/components/BottomNavigationBar.kt

```diff
@@ -0,0 +1,94 @@
+package com.example.pictureupload.ui.screens.landing_screen.components
+
+import androidx.compose.foundation.layout.height
+import androidx.compose.material.BottomNavigation
+import androidx.compose.material.BottomNavigationItem
+import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.example.pictureupload.R
+import com.example.pictureupload.ui.screens.Screens
+
+@Composable
+fun BottomNavigationBar(modifier: Modifier = Modifier) {
+  // val navController = rememberNavController()
+  val iconSize by remember { mutableStateOf(60.dp) }
+  val navBarSize by remember { mutableStateOf(iconSize + 20.dp) }
+  val screens = listOf(
+    Screens.PhotosScreen,
+    Screens.SearchScreen,
+    Screens.SharingScreen,
+    Screens.LibraryScreen,
+  )
+
+  Scaffold(bottomBar = {
+    BottomNavigation(
+      backgroundColor = MaterialTheme.colors.surface,
+      modifier = modifier.height(navBarSize)
+    ) {
+      // val navBackStackEntry by navController.currentBackStackEntryAsState()
+      // val currentDestination = navBackStackEntry?.destination
+
+      screens.forEach { screen ->
+        BottomNavigationItem(
+          icon = {
+            when (screen) {
```

Comment:
We can have icon and label as property of the screen class. This will help us eliminate these when conditions